api/catalog/api/views/media_views.py

```diff
@@ -146,7 +145,7 @@ def report(self, request, *_, **__):
         identifier = media.identifier
         serializer = self.get_serializer(data=request.data)
         if not serializer.is_valid():
-            raise get_api_exception("Invalid input.", 400)
+            raise serializer.is_valid(raise_exception=True)
```

Comment:
This can just be called as a statement as `is_valid` will raise the error on its own. Additionally, we can leave out the conditional as well.

```py
if not serializer.is_valid():
    raise serializer.is_valid(raise_exception=True)
```

Can be completely replaced with just:

```py
serializer.is_valid(raise_exception=True)
```

Reply:
Ok got it @sarayourfriend, I'll make the suggested changes and will make commit of it.